packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -55,7 +59,10 @@ export class Meter implements api.Meter {
     config: MeterConfig = {}
   ) {
     const mergedConfig = merge({}, DEFAULT_CONFIG, config);
-    this._logger = mergedConfig.logger || new ConsoleLogger(config.logLevel);
+    this._diagLogger = getDiagLoggerFromConfig(
+      mergedConfig,
```

Comment:
Unlike other usages of getDiagLoggerFromConfig(), the fallback logic here will still result in a console logger. And the logging level will still use the configured ```config.diagLogLevel``` value as this occurs internally as part of the helper.